### ISSUE TITLE
NuGet package unlisted

### ISSUE THREAD

**Issue by gordon-matt** [StefH/KendoGridBinderEx#47]
Just wondering why the NuGet package has been unlisted. I didn't see any notes as to why in the README. I find it's still working just fine in .NET 6. If there's a security issue or something, please mention it in the README. It would be good to know. Thanks

**Comment by StefH** [comment]
I added this to readme:

n Februari 2022 Progress Software Corporation ("PSC") has obtained from NuGet Prefix reservations for a number of its trademarks/product names, including Kendo.

Since my NuGet packages use the "Kendo" prefix, PSC is hereby requesting that I either rename the package in a way that makes it clear to consumers that this is not a Progress Software Corporation product or delist the package from the nuget.org site altogether.

So for now (Februari 2022) I decided to unlist the NuGet.

When I have time, I'll create new NuGet packages. Probably like:

    GridBinder.for.Kendo (rather than KendoGridBinder)
    GridBinderEx.for.Kendo (rather than KendoGridBinderEx)
    GridBinder.AspNetCore.for.Kendo (rather than KendoGridBinder.AspNetCore)

Kind regards, Stef Heyenrath
